hscontrol/db/node.go

```diff
@@ -349,12 +351,20 @@ func RegisterNodeForTest(tx *gorm.DB, node types.Node, ipv4 *netip.Addr, ipv6 *n
 		panic("RegisterNodeForTest can only be called during tests")
 	}
 
-	log.Debug().
+	logEvent := log.Debug().
```

Comment:
I like this style, we should do it more to clean up very long loglines that make the code hard to read.